src/pages/AboutUs.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography, withStyles } from '@material-ui/core';
+
+import Process from '../components/Process';
+
+const styles = theme => ({
+  centerText: {
+    textAlign: 'center',
+    width: '100%',
+    paddingTop: 20,
+    paddingBottom: 20,
+    marginTop: theme.spacing(10),
+  },
+});
+
+const AboutUs = (props) => {
+  const { classes } = props;
+  return (
+    <React.Fragment>
+      <Typography variant="h5" gutterBottom className={classes.centerText}>Discover our process.</Typography>
+      <Process />
```

Comment:
Components should be used for re-usable entities. The Process component shall not be used anywhere else, and there is no reusability required. So, shift the Process component into the AboutUs page directly.

Reply:
Resolved in commit 97de9971939c449077ef5999058a7b244e275661